### PR TITLE
Add suggested improvements from user feedback

### PR DESCRIPTION
**Pull request by kirch1**
### Files Changed:
- Process.tsx
- App.tsx
- minor css
### What Has Changed: 
- made tweaks based on Chrissy's user feedback session

### CODE REVIEW

**Review by Jesusborjas006 — APPROVED** [review]
Thanks for improving the application from the feedback received!

File: src/Components/Process/Process.tsx [hunk 1]
```diff
@@ -136,10 +135,13 @@ const Process = ({tosInput, setTosInput, setConcerns, setError, user}: FormProps
...
             />
-            <FormGroup row>{getConcernAreaChecks()}</FormGroup>
+            <FormControl sx={{ m: 0 }} component="fieldset" variant="standard">
+              <FormLabel component="legend">Select Area(s) of Concern</FormLabel>
+              <FormGroup row>{getConcernAreaChecks()}</FormGroup>
+            </FormControl>
             <div className="form-footer">
               <FormControl sx={{ m: 0, minWidth: 160 }} size="small">
-                <InputLabel id="tos-library-label">TOS LIbrary</InputLabel>
+                <InputLabel id="tos-library-label">TOS Library</InputLabel>
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:144]: I couldn't believe I hadn't noticed this all the times I looked at it lol

**Review by chrissycooper — APPROVED** [review]

File: src/apicalls.tsx [hunk 2]
```diff
@@ -17,16 +17,17 @@ const processTOS = async (tos: string, concerns: string[], user: number | null):
...
-  //https://tldr-api.onrender.com/api/v1/queries
-  //https://4196c33d-8951-4a3a-8216-bffd37431cc2.mock.pstmn.io/api/v1/processTOS
-  
+
   const response = await fetch(
     "https://tldr-api.onrender.com/api/v1/queries",
     details
   );
 
+  console.log(response)
```
Comment (cjbt) [inline src/apicalls.tsx:26]: make sure you don't push console logs to production

File: src/apicalls.tsx [hunk 3]
```diff
@@ -17,16 +17,17 @@ const processTOS = async (tos: string, concerns: string[], user: number | null):
...
+
   if (!response.ok) {
     const res = await response.json();
+    console.log(res)
```
Comment (cjbt) [inline src/apicalls.tsx:30]: here as well

**Pull request merged**